requirements/base.in

```diff
@@ -1,5 +1,2 @@
 # Core requirements for using this application
 -c constraints.txt
-edx-opaque-keys>=2.3.0
-django>=3.2.15
-django-model-utils>=4.2.0
```

Comment:
Since we're currently targeting two versions of the platform (Nutmeg and Palm) freezing the requirements is making it impossible to pick one or the other, as the version numbers will conflict. These requirements are all satisfied by the platform anyhow, so we can lean on them until we decide to drop support for Nutmeg.